packages/applicant/src/pages/mandatory-questions/[mandatoryQuestionId]/getServerSideProps.test.tsx

```diff
@@ -0,0 +1,274 @@
+import { GetServerSidePropsContext } from 'next';
+import { parseBody } from 'next/dist/server/api-utils/node';
+import {
+  GrantMandatoryQuestionDto,
+  GrantMandatoryQuestionService,
+} from '../../../services/GrantMandatoryQuestionService';
+import {
+  Optional,
+  expectObjectEquals,
+  getContext,
+  mockServiceMethod,
+} from '../../../testUtils/unitTestHelpers';
+import {
+  checkIfPageHaveAlreadyBeenAnswered,
+  generateRedirectUrlForMandatoryQuestionNextPage,
+} from '../../../utils/mandatoryQuestionUtils';
+import { routes } from '../../../utils/routes';
+import getServerSideProps from './getServerSideProps';
+
+jest.mock('next/dist/server/api-utils/node');
+jest.mock('../../../utils/mandatoryQuestionUtils');
+
+const spiedGrantMandatoryQuestionServiceGetMandatoryQuestion = jest.spyOn(
+  GrantMandatoryQuestionService.prototype,
+  'getMandatoryQuestionById'
+);
+
+const spiedGrantMandatoryQuestionServiceUpdateMandatoryQuestion = jest.spyOn(
+  GrantMandatoryQuestionService.prototype,
+  'updateMandatoryQuestion'
+);
+
+describe('getServerSideProps', () => {
+  const getDefaultGrantMandatoryQuestion = (): GrantMandatoryQuestionDto => ({
+    schemeId: 1,
+    submissionId: null,
+    name: null,
+    addressLine1: null,
+    addressLine2: null,
+    city: null,
+    county: null,
+    postcode: null,
+    charityCommissionNumber: null,
+    companiesHouseNumber: null,
+    orgType: null,
+    fundingAmount: null,
+    fundingLocation: null,
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    process.env.USER_TOKEN_NAME = 'gap-test';
+  });
```

Comment:
could set up a beforeEach and afterEach to reset the env var to its original value

like
```
  let userTokenNameBackup;
  beforeEach(() => {
    jest.resetAllMocks();
    userTokenNameBackup = process.env.USER_TOKEN_NAME
    process.env.USER_TOKEN_NAME = 'gap-test';
  });
  afterEach(() => {
    process.env.USER_TOKEN_NAME = userTokenNameBackup;
  });
```